eng/pipelines/preview.yml

```diff
@@ -0,0 +1,74 @@
+trigger: none
+
+pr: none
+
+parameters:
+- name: UseTypeSpecNext
+  displayName: Use TypeSpec "next" version
+  type: boolean
+  default: true
+
+resources:
+  repositories:
+    - repository: sdk-repository
+      type: github
+      endpoint: Azure
+      name: Azure/azure-sdk-for-python
+      ref: refs/heads/users/pahallis/cookiecutter
```

Comment:
nit: is this going to stay in the code pushed to main?

Reply:
It won't.  I can change it now or wait until https://github.com/Azure/azure-sdk-for-python/pull/32011 merges